src/connectors/configure/connectConfigure.js

```diff
@@ -0,0 +1,73 @@
+import { checkRendering } from '../../lib/utils.js';
+
+const usage = `Usage:
+var configure = connectConfigure(function render(params, isFirstRendering) {
+  // params is whatever you put as argument to the widget
+  // usually you'll just have a function which does nothing
+  // as rendering for connectConfigure
+});
```

Comment:
I think it might be confusing for users to talk about "render" since this connector should not render anything?
For instance in react-instantsearch the connector is not even exposed to end-users and the Configure component just return `null`